design-toolkit/components/src/components/options-item/index.tsx

```diff
@@ -102,10 +102,11 @@ export function OptionsItem<T extends IOptionsItem>({
 
         return (
           <>
-            <span className='flex w-[16px] items-center'>
-              {prefixIcon && <Icon>{prefixIcon}</Icon>}
-            </span>
-
+            {prefixIcon && (
```

Comment:
once this component is refactored to use twv, is the plan to make this a a slot with the style of `empty:none` ? or keep the conditional jsx

Reply:
Haven't given it much thought yet